modules/op-configuring-custom-sa-and-scc.adoc

```diff
@@ -89,8 +90,10 @@ subjects:
 
 <2> Define a custom SCC created based on restricted privileges, with modified `runAsUser` field.
 
-<3> Restrict any pod that gets attached with the custom SCC through the custom SA to run as user id `1000`.
+<3> At this time, enabling the `allowPrivilegeEscalation` setting is required for Buildah to run successfully in the container. With this setting, Buildah can leverage `SETUID` and `SETGID` capabilities when running as a non-root user.
```

Comment:
I'd consider this being an IMPORTANT admonition. 
(Advisory information essential to the completion of a task. Users must not disregard this information.)

Reply:
you mean just put IMPORTANT: in the text? I don't think a note block would work with a callout